server/pedersen_ec.go

```diff
@@ -51,7 +50,7 @@ func (s *Server) PedersenEC(curveType groups.ECurve, stream pb.Protocol_RunServe
 		return err
 	}
 
-	el := types.ToECGroupElement(ecgrop)
+	el := ecgrop.GetNativeType()
```

Comment:
Why not ecgroup? Or perhaps ecGroup?

Reply:
:smile:  I have no idea, it's been like that for a while. I changed the names to to `el` (pb type) and `myEl` (emmy type) so that its similar as with `h` and `myH` above.